hummingbot/connector/markets_recorder.py

```diff
@@ -269,10 +273,10 @@ def append_to_csv(self, trade: TradeFill):
             age = pd.Timestamp(int(trade.timestamp / 1e3 - int(trade.order_id[-16:]) / 1e6), unit='s').strftime('%H:%M:%S')
         if not os.path.exists(csv_path):
             df_header = pd.DataFrame([["Config File", "Strategy", "Exchange", "Timestamp", "Market", "Base", "Quote",
-                                       "Trade", "Type", "Price", "Amount", "Fee", "Age", "Order ID", "Exchange Trade ID"]])
+                                       "Trade", "Type", "Price", "Amount", "Leverage", "Fee", "Age", "Order ID", "Exchange Trade ID", "Position"]])
             df_header.to_csv(csv_path, mode='a', header=False, index=False)
-        df = pd.DataFrame([[trade.config_file_path, trade.strategy, trade.market, trade.timestamp, trade.symbol, trade.base_asset, trade.quote_asset,
-                            trade.trade_type, trade.order_type, trade.price, trade.amount, trade.trade_fee, age, trade.order_id, trade.exchange_trade_id]])
+        df = pd.DataFrame([[trade.config_file_path, trade.strategy, trade.market, trade.timestamp, trade.symbol, trade.base_asset, trade.quote_asset, trade.trade_type,
+                            trade.order_type, trade.price, trade.amount, trade.leverage, trade.trade_fee, age, trade.order_id, trade.exchange_trade_id, trade.position]])
```

Comment:
Watch out for latest push modifying the trade fills csv export. With the new code, it would pick up the new leverage column automatically from the TradeFill object

Reply:
Github should handle it or make me resolve conflict before merge.

Reply:
Seems there's a conflict already. I'll resolve first thing today.